tests/integration/cloudformation/test_cloudformation_iam.py

```diff
@@ -125,3 +127,25 @@ def test_policy_attachments(
     policy = matching[0]["AssumeRolePolicyDocument"]
     policy = json.loads(policy) if isinstance(policy, str) else policy
     assert policy["Statement"][0]["Principal"] == {"Service": "elasticbeanstalk.amazonaws.com"}
+
+
+@pytest.mark.aws_validated
+def test_iam_username_defaultname(deploy_cfn_template, iam_client, snapshot):
+    snapshot.skip_key(re.compile("UserId"), "<user-id>")
```

Comment:
nit: wondering if `*Id` attributes could also become part of the default `skip_key` list in `snapshots/prototype.py` (similar to `*Name` attributes)..? Not related to the PR per se, just out of curiosity - I guess we'll figure that out as we go 👍

Reply:
>  I guess we'll figure that out as we go

That'd be my preferred approach here :100:  Trying to work a bit "data-driven" here and get some samples first before generalizing too much.